python/fledge/services/core/api/auth.py

```diff
@@ -326,6 +326,13 @@ async def get_user(request):
             u["accessMethod"] = user.pop('access_method')
             u["realName"] = user.pop('real_name')
             u["description"] = user.pop('description')
+            block_until = user.pop('block_until')
+            if block_until:
+                curr_time = datetime.datetime.now(datetime.timezone.utc).strftime(DATE_FORMAT)
+                block_time = block_until.split('.')[0] # strip time after HH:MM:SS for display
+                if datetime.datetime.strptime(block_until, DATE_FORMAT) > datetime.datetime.strptime(curr_time, DATE_FORMAT):
+                    u["status"] = "blocked"
+                    u["blockUntil"] = block_time
```

Comment:
For user profile we do not need `status and blockUntil` KV pair's, this can be removed from here.